lib/vbox/machine.go

```diff
@@ -245,18 +246,36 @@ func download(path string, repository repo.Repository) {
 	}
 }
 
-// CheckDeps checks for virtualbox dependencies
-func CheckDeps(pkg string) error {
-	err := exec.Command("which", pkg).Run()
-	if err != nil {
-		log.Error("Error while running `which` : ", err.Error())
-		return fmt.Errorf("[-] Could not find virtualbox, please install virtualbox")
+// CheckVBInstalled checks for virtualbox dependencies
+func CheckVBInstalled() error {
+	vbm := "VBoxManage"
+	if err := exec.Command("which", vbm).Run(); err != nil {
+		if runtime.GOOS == "windows" {
+			path := `C:\Program Files\Oracle\VirtualBox\`
+			if !help.DirExists(path) {
+				path = dialogs.GetSingleAnswer("[-] Couldn't find VirtualBox in default location, please specify installation dir manually: ", dialogs.EmptyStringValidator)
+				if help.DirExists(path) {
+					return fmt.Errorf("Could not find virtualbox, may be you need to install it first")
+				}
+			}
+			exec.Command("setx", "path", `%path%;`+path).Run()
+			if err := exec.Command("PowerShell", "-Command", "[Environment]::SetEnvironmentVariable('Path', $env:Path + ';"+path+"', [EnvironmentVariableTarget]::User)").Run(); err != nil {
+				log.Error("Error setting PowerShell `path` : ", err.Error())
+				return fmt.Errorf(`Couldn't alter your system PATH, please change it manually. E.g. 'C:\Program Files\Oracle\VirtualBox\'`)
+			} else {
```

Comment:
__[golint]__
* [golint] if block ends with a return statement, so drop this else and outdent its block (move short variable declaration to its own line if necessary)

<!-- Comment by [SideCI](https://sideci.com) -->